File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/template_ns_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'template_ns'

include Template_ns

p1 = Pairii.new(2, 3)
p2 = Pairii.new(p1)

raise RuntimeError if p2.first != 2
raise RuntimeError if p2.second != 3

p3 = Pairdd.new(3.5, 2.5)
p4 = Pairdd.new(p3)

raise RuntimeError if p4.first != 3.5
raise RuntimeError if p4.second != 2.5

File: support_tools/swigwin-3.0.12/Examples/test-suite/python/template_ns_runme.py
from template_ns import *
p1 = pairii(2, 3)
p2 = pairii(p1)

if p2.first != 2:
    raise RuntimeError
if p2.second != 3:
    raise RuntimeError

p3 = pairdd(3.5, 2.5)
p4 = pairdd(p3)

if p4.first != 3.5:
    raise RuntimeError

if p4.second != 2.5:
    raise RuntimeError
